### PR TITLE
lugaru: init at 1.2

### PR DESCRIPTION
**Pull request by bignaux**
###### Motivation for this change

OSS Lugaru !

###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nix-review --run "nix-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Assured whether relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

File: pkgs/games/lugaru/default.nix [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+with stdenv.lib;
+
+stdenv.mkDerivation rec {
+
+  pname = "lugaru";
+  version = "1.2";
+
+  src = fetchurl {
+  	url = "https://gitlab.com/osslugaru/lugaru/-/archive/${version}/lugaru-${version}.tar.bz2";
```
Comment (flokli) [inline pkgs/games/lugaru/default.nix]: there's a [tab] here that shouldn't.

**Review by flokli — CHANGES_REQUESTED** [review]

File: pkgs/games/lugaru/default.nix (hunk 1 above)
Comment (bignaux) [reply pkgs/games/lugaru/default.nix]: fixed

File: pkgs/games/lugaru/default.nix [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+  libvorbis, SDL2, makeWrapper, libpng, libjpeg_turbo, libGLU }:
+
+with stdenv.lib;
+
+stdenv.mkDerivation rec {
+
+  pname = "lugaru";
+  version = "1.2";
+
+  src = fetchurl {
```
Comment (flokli) [inline pkgs/games/lugaru/default.nix]: please use `fetchfromgitlab`, if there's no specific reason not to.

Comment (bignaux) [reply pkgs/games/lugaru/default.nix]: done . i don't know there is such thing for gitlab ...

**Comment by flokli** [comment]
@GrahamcOfBorg build lugaru

**Pull request merged**